inventory/src/main/java/org/flyve/inventory/categories/Cameras.java

```diff
@@ -67,20 +85,31 @@
 	public Cameras(Context xCtx) {
         super(xCtx);
 
+        cameraVendors = Utils.loadJSONFromAsset(xCtx, "camera_vendors.json");
```

Comment:
@ivans51 When the vendor name of the camera is not in the JSON, do you print the ID, right?

Reply:
The vendor name of the manufacturer is obtained from the model camera is pass as id within of the JSON file, if not found print N/A

Reply:
Show the **name** if the **ID** exist in the JSON data.
Show the **ID** if the **name** is not name in the JSON.
Show **N/A** when no **ID** from the camera.

Reply:
In the second case no is possible because in Android no is possible get this value (name) but if the model camera is the ID of the JSON